velox/dwio/parquet/tests/writer/ParquetWriterTest.cpp

```diff
@@ -22,7 +22,9 @@
 #include "velox/connectors/hive/HiveConnector.h" // @manual
 #include "velox/core/QueryCtx.h"
 #include "velox/dwio/parquet/RegisterParquetWriter.h" // @manual
+#include "velox/dwio/parquet/reader/PageReader.h"
 #include "velox/dwio/parquet/tests/ParquetTestBase.h"
+#include "velox/dwio/parquet/writer/arrow/Properties.h"
```

Comment:
This header can be removed when we add a new enum.

Reply:
Removed.